test-with-jest/src/mock-function/data/fake-data.ts

```diff
@@ -0,0 +1,28 @@
+import { v4 as uuidv4 } from 'uuid';
+import { faker } from '@faker-js/faker';
```

Comment:
I definitely get the appeal but for a simple example on how to test azure mocks I recommend just putting placeholder values like "firstName", "lastName" or using whatever "personas" have been created for contoso

A fun example of why we avoid 3p dependencies where possible https://www.bleepingcomputer.com/news/security/dev-corrupts-npm-libs-colors-and-faker-breaking-thousands-of-apps/